### DISCUSSION TITLE
Is it possible to export dust3d poses?

### DISCUSSION THREAD

**Discussion by cgparkhouse** [huxingyi/dust3d#117] [General]
I'm new to Dust3d, but there seems little documentation on Poses.  What I want to do is to set a pose and then export the model as an image.

**Comment by huxingyi** [comment]
Sorry, the poses feature will be removed in the future release.

**Comment by cgparkhouse** [comment]
I am sure it will make sense in time.  I used the .fix file today in
Buildbox - works great but both motions I created  have the same name

Also I use Mac and not
Having a middle mouse button makes it impossible to move the model - can
you add an option with a key - space bar - and left button to move the
model???

Dust3d is really cool - well done

**Reply by huxingyi** [reply]
Thanks. You can try hold alt key and use left button to rotate the model, and hold alt+shift key to move.

**Comment by cgparkhouse** [comment]
Many thanks - sorry I missed that in the documentation

All the very best and thanks for getting back to me.

Chris

**Comment by cgparkhouse** [comment]
Hi Jeremy

Has any testing been done on Apple Silicon using Rosetta 2?

I am planning on buying an M1 Mac soon and it seems that Dust3d is the only
App that I use that does not have native support.

Very best regards

Chris

**Reply by huxingyi** [reply]
Not yet.

**Comment by cgparkhouse** [comment]
Here is a link to a guy who ported GIMP to M1.  It sounds quite
straightforward...

https://www.reddit.com/r/MacOS/comments/mb5ekq/unofficial_m1_compatible_gimp_build/

Good luck

**Comment by cgparkhouse** [comment]
Sorry to bother you again - I am trying to understand the Materials section
and what each file upload option means (Base Color, Normal Map, Metalness,
Roughness, Ambient Occlusion).  I've search the web for PBR, but so far
have not found any useful info to help me.  The problem I am facing is that
in my model I am getting a light cyan showing where it should be blue...
Attached photo of my test model (based on you excellent Giraffe). Thanks
for your great work.

**Comment by cgparkhouse** [comment]
I was trying to move a set of lines I had drawn and after selecting them
all and dragging from the right to the left dust3d crashed (I liked the
fact that it auto saved and next time I opened dust3d it opened the file
and I was able to save it)

I repeated this 3 times with the same crash.

Please find attached the file that crashes

MacBook Pro 2015, 15" i7 16GB RAM, 2TB SSD.
macOS  11.5.1
dust3d 1.0.0.36

In gratitude

Chris
